<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="samsung-PC" ignored="0" name="failed_tc_retry" tests="12" failures="3" timestamp="2020-05-14T21:56:50 EDT" time="0.41" errors="0">
  <testcase name="form" time="0.047" classname="RetryAnalyzer.MyTests">
    <skipped/>
  </testcase> <!-- form -->
  <testcase name="form" time="0.005" classname="RetryAnalyzer.MyTests">
    <skipped/>
  </testcase> <!-- form -->
  <testcase name="form" time="0.01" classname="RetryAnalyzer.MyTests">
    <skipped/>
  </testcase> <!-- form -->
  <testcase name="form" time="0.005" classname="RetryAnalyzer.MyTests">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at RetryAnalyzer.MyTests.form(MyTests.java:23)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- form -->
  <testcase name="login" time="0.005" classname="RetryAnalyzer.MyTests">
    <skipped/>
  </testcase> <!-- login -->
  <testcase name="login" time="0.005" classname="RetryAnalyzer.MyTests">
    <skipped/>
  </testcase> <!-- login -->
  <testcase name="login" time="0.005" classname="RetryAnalyzer.MyTests">
    <skipped/>
  </testcase> <!-- login -->
  <testcase name="login" time="0.005" classname="RetryAnalyzer.MyTests">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at RetryAnalyzer.MyTests.login(MyTests.java:11)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase name="reg" time="0.008" classname="RetryAnalyzer.MyTests">
    <skipped/>
  </testcase> <!-- reg -->
  <testcase name="reg" time="0.005" classname="RetryAnalyzer.MyTests">
    <skipped/>
  </testcase> <!-- reg -->
  <testcase name="reg" time="0.005" classname="RetryAnalyzer.MyTests">
    <skipped/>
  </testcase> <!-- reg -->
  <testcase name="reg" time="0.005" classname="RetryAnalyzer.MyTests">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at RetryAnalyzer.MyTests.reg(MyTests.java:17)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- reg -->
</testsuite> <!-- failed_tc_retry -->
